tests/integration_tests/test_integration_dds.py

```diff
@@ -102,6 +102,9 @@ def test_can_stage_and_delivery_project_dir(self):
 
                 status_response = yield self.http_client.fetch(delivery_link)
                 self.assertEqual(json.loads(status_response.body)["status"], DeliveryStatus.delivery_skipped.name)
+                
+                dds_version = delivery_resp_as_json['dds_version']
+                self.assertGreater(len(dds_version), 0, "DDS version should not be empty")
```

Comment:
it would be good to assert that it has parsed the correct version string (i.e. `2.6.1`), not just that the string is non-empty

Reply:
I think I changed the dds_version to return the string `Data Delivery System, version 2.6.1` instead of just `2.6.1.`
This is cause I changed from using `subprocess.run `to using the _run() method that would actually run dds --version

And I didn't know the `type` returned after running `dds --version ` so as to split and get the version only

Should I look deeper into this so that the dds_version returned by arteria_delivery is actually the value itself i.e 2.6.1 and not the whole string i.e Data Delivery System, version 2.6.1?